includes/managers/controls.php

```diff
@@ -301,8 +301,11 @@ class Controls_Manager {
 	 *
 	 * @var array
 	 */
+	// TODO: Remove in the future. - When Pro will remove the template class use in location manager.
```

Comment:
It's a private property, and it's safe to remove it

Reply:
In pro version, there is a use in :

https://github.com/elementor/elementor-pro/blob/c35e2084b5884c3b92de9da3817655a0c10e2432/modules/theme-builder/classes/locations-manager.php

of 

https://github.com/elementor/elementor-pro/blob/6ab8d9dc94328c75ba80396c64992fe51dd63137/modules/theme-builder/files/css/template.php

with the next function 

https://github.com/elementor/elementor/blob/3994d6d3c801e2234f60a0f150630061fd54f20c/includes/managers/controls.php#L894

That has support only for one time flag value update per page render per document.
So I need to support both ways until we will remove the support from the Pro version.
Then the location manager in pro will inherit the new logic from base that support multipal page ID's support.

https://github.com/maork-elementor/elementor/blob/b36f98759cad724e413d10200400ae1bb2aabd89/includes/managers/controls.php#L906

Reply:
It can't be an external direct use of a private property.

Reply:
In my understanding, we will continue to support each other in both directions?
I have removed the comment.